src/client/pfs_new_storage.go

```diff
@@ -128,7 +129,35 @@ func (c APIClient) GetTarConditional(repoName string, commitID string, path stri
 		}
 	}
 	return nil
+}
 
+// ListFileV2 streams FileInfoNewStorage similar to ListFileF
```

Comment:
We probably shouldn't reference functions that are going to be removed. `ListFileV2` will subsume the `ListFileF` functionality.